charts/fybrik/values.yaml

```diff
@@ -445,6 +487,7 @@ opaConnector:
 
   # Overrides the URL of the OPA server
   # Defaults to the address of the deployed OPA server
+  # If the OPA server is configured with TLS then use https://opa:443
```

Comment:
does OPA use 443 ?

Reply:
its not offical chart but rather an example: https://github.com/helm/charts/blob/master/stable/opa/values.yaml